queue/queue_header.php

```diff
@@ -1,9 +1,14 @@
+<script src="http://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
+<script src="/queue/queue_header.js"></script>
 <div class="container-fluid dashboard bg-light py-3" ng-cloak>
 	<div class="d-flex flex-row justify-content-between align-items-center">
 		<div class="d-flex flex-row">
 			<div class="queue-size-lbl">Queue:</div>
 			<div class="queue-size-count">{{appointments.length}}</div>
 		</div>
+		<select class="btn dropdown-toggle" id="siteSelect" ng-model="selectSite">
+			<option value="">Select A Site</option>
```

Comment:
Currently this default option is selectable and shows up in the drop-down options. ```style="display:none"``` will make it not show up in the drop-down.
e.g. ```<option value="" style="display:none"> -- select a site -- </option>```